protocol/proto/devices.proto

```diff
@@ -31,5 +31,4 @@ enum DeviceType {
     UNKNOWN_SPOTIFY = 100;
     CAR_THING = 101;
     OBSERVER = 102;
-    HOME_THING = 103;
```

Comment:
I would like to keep the protobuf as is. Modifying it would make updating it a bit inconvenient and possibly break the change when someone doesn't know that it was removed intentionally^^

In code you could use the [`unimplemented!`](https://doc.rust-lang.org/std/macro.unimplemented.html) macro to communicate the missing support for example.

Reply:
about `unimplemented!`: I tried adding 
```
"homething" => unimplemented!()
```
to `()impl FromStr for DeviceType` but that results in a panic when calling the binary with `--device-type=homething`
Which is probably not what we want.  Sorry, but I am not sure where I should signal that this is unimplemented intentionally

Reply:
Yeah unimplemented is pretty much just an intentional panic. But if we can just handle it as error that's probably better.